src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx

```diff
@@ -61,38 +167,157 @@ If you send span events as described by the [OpenTelemetry specification](https:
 If you have span events, links for these appear in the right pane:
 
 1. Click on a span in the waterfall view.
-2. In the right pane, click **View span events**, or expand **Error details** and click the link at the bottom of the errors.
+2. In the right pane, click **View span events**, or you can expand **Error details** and click the link at the bottom of the errors.
 
   ![Screenshot showing the right pane showing the two links for span events](./images/view-span-events.png "Screenshot showing the right pane showing the two links for span events")
 
-3. When you're in span events and only want to view exceptions, toggle **Only show exceptions**.
+3. When you're in span events and only want to view exceptions, slide the toggle **Only show exceptions**.
 
 ![Screenshot showing span events and how you can filter just for exceptions](./images/span-events-exceptions.png "Screenshot showing span events and how you can filter just for exceptions")
             
 <Callout variant="tip">
-OpenTelemetry exceptions handled by the app/service are displayed independently of span error status. These exceptions are not necessarily associated with a span error status.
+OpenTelemetry exceptions handled by the app/service are displayed independently of span error status and are not necessarily associated with a span error status.
 </Callout>
 
-### Transactions [#trx]
-
-Use **Transactions** to identify slow or error transactions that might be causing a spike in your application's response time. To get a list of transactions: From the **Transaction Summary** page, select the transactions table.
-
 ### Databases [#databases]
 
 The **Databases** page shows an application's database and cache data. The page shows individual database transactions as a sortable table, and shows operations, throughput, and response time as charts.
 
+For your data to appear in this section, make sure it has the following:
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "200px" }}>
+        UI area
+      </th>
+      <th>
+        Attribute
+      </th>
+    </tr>
+  </thead>
+    <tbody>
+    <tr>
+      <td>
+        Top database calls
+      </td>
+      <td>
+        * `span.kind = client` or `producer`
+        * `db.system`
+        * Facets by span name (should follow DB semantic conventions - [TO_DO_FIND_LINK and CLARIFY span.name!])
```

Comment:
Is this your TO DO?

Reply:
@mmfred Now, that was embarrassing! I'm removing that!